tests/pytests/test_index_oom.py

```diff
@@ -237,56 +252,11 @@ def test_change_config_during_bg_indexing(env):
   memory_ratio = used_memory / max_memory
   env.assertAlmostEqual(memory_ratio, 0.8, delta=0.1)
 
-@skip(cluster=True)
```

Comment:
I would leave this test if we can make it run. if not comment it out and make it run in the future. I think it is important not to delete it

Reply:
Unless we plan to change the behavior in the near future, I don't see a reason to leave this test commented...